DiffusionPreprocessing/scripts/eddy_postproc.sh

```diff
@@ -87,13 +87,30 @@ else
 fi
 #fi
 
+
+# Create a mask representing voxels within the field of view for all volumes prior to dilation
+# 'eddy' can return negative values in some low signal locations, so use -abs for determining the fov mask
+${FSLDIR}/bin/fslmaths ${datadir}/data -abs -Tmin -bin -fillh ${datadir}/fov_mask
+
 	 
 if [ ! $GdCoeffs = "NONE" ] ; then
     echo "Correcting for gradient nonlinearities"
 	# Note: "data_warped" is eddy-current and suspectibility distortion corrected (via 'eddy'), but prior to gradient distortion correction
 	# i.e., "data_posteddy_preGDC" would be another way to think of it
     ${FSLDIR}/bin/immv ${datadir}/data ${datadir}/data_warped
-    ${globalscriptsdir}/GradientDistortionUnwarp.sh --workingdir="${datadir}" --coeffs="${GdCoeffs}" --in="${datadir}/data_warped" --out="${datadir}/data" --owarp="${datadir}/fullWarp"
+
+    # Dilation outside of the field of view to minimise the effect of the hard field of view edge on the interpolation
+	  DiffRes=`${FSLDIR}/bin/fslval ${datadir}/data pixdim1`
+    DilateDistance=`echo "$DiffRes * 4" | bc`  # Extrapolates the diffusion data up to 4 voxels outside of the FOV
+    ${CARET7DIR}/wb_command -volume-dilate ${datadir}/warped/data_warped.nii.gz $DilateDistance NEAREST ${datadir}/warped/data_warped_dilated.nii.gz
```

Comment:
Here's the bug that I was referring to.  (I guess I forgot to actually "Add" the comment at the specific line).  Namely, in `eddy_postproc.sh` the "warped" directory doesn't exist at this point in the script.  This line should be:
`${CARET7DIR}/wb_command -volume-dilate ${datadir}/data_warped.nii.gz $DilateDistance NEAREST ${datadir}/data_warped_dilated.nii.gz`

Reply:
fixed